doc/deployment/deploy-to-aks-with-acr-and-git-using-jenkins.md

```diff
@@ -0,0 +1,140 @@
+# Deploy to Azure Kubernetes Service using Jenkins
+
+This document shows how to deploy this todo app java project to Kubernetes cluster using Jenkins.
```

Comment:
Let's include an intro paragraph to distinguish the two building from local and GitHub flows. So may be mention this. "You can queue an ACR quick build with local code . This document shows you how you can modify an existing CI/CD pipeline so that a JAR file built in Jenkins is uploaded to ACR together with the Dockerfile to produce the Docker image that will be deployed to AKS."